config/staging.js

```diff
@@ -20,5 +20,6 @@ module.exports = {
 	jwt: {
```

Comment:
Is this where we can configure the jwt expiration time? (related #200)

Reply:
Not sure, tbh, I think the problem may be more nuanced because we want to refresh the token not just increase the expiry.